.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.defaultInterpreterPath": "/workspaces/rosette/python/.venv/bin/python"
```

Comment:
@nivanov-ati this is what does it - it wouldn't work in a monorepo with multiple independent Python codebases, but its a suitable fix for here.